### DISCUSSION TITLE
Custom flash msg

### DISCUSSION THREAD

**Discussion by mateusgalasso** [laravel/nova-issues#4355] [Q&A]
I would like to know if there is any way before the redirect sends a message to Laravel Nova's toast

**Comment by crynobone** [comment]
which redirect?

On Resource Created you could do:

```js
Nova.booting(() => {
  Nova.$on('resource.created', () => Nova.success('Yey!'))
})
```

**Comment by mateusgalasso** [comment]
No, 
I'd like something like
`return redirect(route('nova.pages.index', ['resource' => 'proposals']));`
and passing msg to session so, on the next page I can show this msg to the user

**Reply by crynobone** [reply]
You should use implements custom flash messages based on <https://inertiajs.com/shared-data#flash-messages>
